### PR TITLE
Remove query params from url

### PR DESCRIPTION
**Pull request by vipulbhj**
#21 fixed

please review when you get time

### CODE REVIEW

File: src/lib/utils.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default class Utils {
+  urlWithoutQueryParameters(url) {
```
Comment (fxnoob) [inline src/lib/utils.js]: Put jsdoc comments here

File: option-page/components/index.jsx [hunk 2]
```diff
@@ -33,4 +33,4 @@ export default class Index extends React.Component {
 
     return <React.Fragment>{html}</React.Fragment>;
   }
-}
```
Comment (fxnoob) [inline option-page/components/index.jsx]: Revert this change

File: option-page/option.html [hunk 3]
```diff
@@ -2,7 +2,6 @@
 <html>
 <head>
 </head>
-
```
Comment (fxnoob) [inline option-page/option.html]: Revert this change

File: src/background.js [hunk 4]
```diff
@@ -50,14 +52,16 @@ class Main extends ChromeApi {
    * @memberof Main
    */
   performURLChecks = async (details, callback) => {
-    const { url, tabId } = details
+    const { url, tabId } = details;
+
+    const modifiedUrl = utils.urlWithoutQueryParameters(url);
```
Comment (fxnoob) [inline src/background.js]: Change modifiedUrl to urlWithoutQueryParam

**Comment by vipulbhj** [comment]
Okay, working on it

**Comment by vipulbhj** [comment]
@fxnoob made the suggested changes. Please take a look

File: src/lib/utils.js (hunk 1 above)
Comment (fxnoob) [reply src/lib/utils.js]: Pls take help from chromeApi.js for jsdoc commenting

File: option-page/components/index.jsx (hunk 2 above)
Comment (fxnoob) [reply option-page/components/index.jsx]: Revert this change

**Comment by vipulbhj** [comment]
That is just a space change, it is correctly indented now. 

Looking in jsdoc

**Comment by fxnoob** [comment]
For indentation we can integrate prettier in separate issue

**Comment by vipulbhj** [comment]
yes we can do that.

If everything looks good, please merge this
